addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -563,6 +574,27 @@ public function store
                }
             }
 
+            // Create options
+            $optionsIds = array();
+            foreach ($request->options as $optionValue) {
```

Comment:
I have discussed this section with dia, We don't need to an extra query here. The ads already has the up to date info.

Reply:
I added 2 queries, one to delete the deleted options (if any) and the other is to create the new options (if any)